include/filewriter/summarywriter_impl.hpp

```diff
@@ -200,9 +321,10 @@ void SummaryWriter<Filewriter>::PRCurve(const std::string& tag,
         // Include the exact number in previous bucket.
         if (*lb != item)
             lb--;
-        truePositives[lb - edges.begin()] = truePositives[lb - edges.begin()] + (v * weights[i]);
-        falsePositives[lb - edges.begin()] = falsePositives[lb - edges.begin()] +
-            ((1 - v) * weights[i]);
+        truePositives[lb - edges.begin()] = truePositives[lb - edges.begin()] +
+            (v * weights[i]);
+        falsePositives[lb - edges.begin()] = falsePositives[lb - edges.begin()]
+            + ((1 - v) * weights[i]);
```

Comment:
```suggestion
        truePositives[lb - edges.begin()] = truePositives[lb - edges.begin()] +
            (v * weights[i]);
        falsePositives[lb - edges.begin()] = falsePositives[lb - edges.begin()]
            + ((1 - v) * weights[i]);
```

You have used four space tab in this function please do it 2 space tab.